pkg/nodebase/NodebaseApp.js

```diff
@@ -8,9 +8,9 @@
  */
 import {App} from '../Library/App/Worker/App.js';
 import {RecipeService} from '../Library/Arcs/RecipeService.js';
-import {StoreUpdateService} from '../Library/Arcs/StoreUpdateService.js';
+import {StoreService} from '../Library/Arcs/StoreService.js';
```

Comment:
Store and Recipe services are built into ArcsWorker, you don't want them here.

Reply:
removed